### ISSUE TITLE
Problematic file names

### ISSUE THREAD

**Issue by danielmlow** [ReproNim/reproschema-ui#263]
@sanuann , the files that submitted at the end of a protocol are currently saved like this (bottom files in white): 

![image](https://user-images.githubusercontent.com/14787791/135877790-b81343d1-91b2-46e4-be26-6fcf44ec6faa.png)

having characters after `.zip` can cause problems for some people when trying to unzip. could we change it to 

test_debbie_01-001.zip
test_debbie_01-002.zip
test_debbie_01-003.zip
test_debbie_01-004.zip
etc?

So that they end in `.zip` and are easy to unzip?

**Comment by sanuann** [comment]
i don't remember why exactly it was done that way. @satra is it okay to change it to the format daniel mentioned above?

**Comment by satra** [comment]
isn't that the multi part upload? in which case the zip file should be constructed by concatenating the parts.

**Comment by satra** [comment]
one should not be trying to unzip those files, just concatenating them. and the multipart upload should also apply potentially to activity uploads no?

**Comment by sanuann** [comment]
yes, they are the multipart uploads. for the activity uploads, I chose not to use multipart upload. do we need it by default for any uploads?

**Comment by danielmlow** [comment]
other people using reproschema in the future may not be as tech savvy, so that’s why i think it’s worth changing it so that it is easy to unzip or combine.
